bot/exts/evergreen/emoji.py

```diff
@@ -67,8 +69,16 @@ def generate_invalid_embed(emojis: list) -> Tuple[discord.Embed, List[str]]:
         msg.append(f"These are the valid categories\n```{error_comp}```")
         return embed, msg
 
-    @commands.command(name="emojicount", aliases=["ec", "emojis"])
-    async def emoji_count(self, ctx: commands.Context, *, category_query: str = None) -> None:
+    @commands.group(name="emoji", invoke_without_command=True)
+    async def emoji_group(self, ctx: commands.Context, emoji: Optional[Emoji]) -> None:
+        """A group of commands related to emojis."""
+        if emoji is not None:
```

Comment:
```suggestion
        if emoji:
```

Reply:
If the user doesn't input a name for this argument, `emoji` will be explicitly `None`. I think it's better to say that's what we're checking for.

Reply:
Well `if emoji:` is the same as `if emoji is not None:`. And according to me `if emoji:` seems the better way

Reply:
Let's avoid arguing over code style opinions unless the style is clearly defined in our projects. 

Since both works, and there's not really an explicit bias in our projects other than a slight lean towards "explicit is better than implicit", there's no need to request a change for this.